### ISSUE TITLE
decomposition of 4-body operator spin-free case

### ISSUE THREAD

**Issue by DanielCalero1** [sokolov-group/sqa_plus#7]
I wonder if you have worked on the implementation of the function "decomp_4op_to_2op_2rdm_sf", which in the original version of sqa says is not yet implemented.

I looked into your main branch and I saw that at least is not implemented there, but I wonder if you have worked in it somewhere else.

**Comment by ssh2** [comment]
No, we do not need the operator decomposition capabilities for our purposes, so we haven't work on this.
